experimental/sql/dialects/rel_impl.py

```diff
@@ -585,7 +585,7 @@ class Project(Operator):
   def get(input: Operation, res_names: List[str], res_types: List[DataType],
           projection: Region) -> 'Project':
     return Project.create(operands=[input.result],
-                          result_types=[Bag.get(res_names, res_types)],
```

Comment:
This is an unrelated change. I can remove it, if you prefer.

Reply:
Leave it as is now but I prefer separate PRs for the future.